scripts/flamegraph.py

```diff
@@ -125,12 +125,7 @@ def format_details(self, name, events):
             for event in events:
                 if event.startswith("tls::ext::"):
                     details.append(event[len("tls::ext::"):])
-        elif name.startswith("tls::certificate_"):
```

Comment:
used to have two branches, now it has none, why?

Reply:
True, the second one was dead code in the old version. But why are there zero branches for `tls::certificate_` in the new version?